libzeth/circuits/binary_operation.hpp

```diff
@@ -119,6 +118,7 @@ class double_bit32_sum_eq_gadget : public libsnark::gadget<FieldT>
 };
 
 } // namespace libzeth
+
```

Comment:
I've switched to having a space here consistently across all files.  It seems more consistent with the rest of the coding standards (we always leave an empty line after closing a function or namespace, and always have an empty line before includes), and matches the layout in libsnark.

Reply:
Sounds good @dtebbs (as long as we're consistent everywhere that's fine)